server/tests/test_notebook_view.py

```diff
@@ -151,6 +152,43 @@ def test_tryit_view(client, fake_user, logged_in, iomd):
         assert len(response.redirect_chain) == 0
 
 
+@pytest.mark.parametrize("file_size", [1, 10485764])
+def test_from_template_view(client, file_size):
+    endpoint = "from-template"
+
+    # endpoint should not accept a GET request
+    resp = client.get(
+        reverse(endpoint)
+    )
+    assert resp.status_code == 405
```

Comment:
Each of these should be separate test cases, it makes it easier to debug what's working / what isn't when there are changes